app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
```

Comment:
- [ ] For us to successively create a post, the PostsController should have `create` :+1: and `new` :-1: actions as the requirements say. Please kindly add a new action and moves a step toward approval. It should be something similar to this: 

```ruby
    def new
      @post = Post.new
    end
```


- [ ] Also, make sure that you create a `new.html.erb` file with a form in the directory views/posts/ which should accept inputs for creating posts. 

Please kindly address the above issues :pray: